discovery-provider/es-indexer/etl/indexNames.ts

```diff
@@ -0,0 +1,9 @@
+export const indexNames = {
+  follows: 'follows1',
```

Comment:
what's the 1 about?

Reply:
Version number suffix... when we make a mapping change we roll to `follows2` both in the indexer and in the python code.

Eventually we can add some code to delete version `n-2` so when we get to `follows3` we can delete `follows1`.

This ensures old code and new code can run on same ES instance without any problems... which is good for rolling deploy type of stuff.